src/software/ai/hl/stp/tactic/dribble/dribble_fsm.h

```diff
@@ -214,4 +214,5 @@ struct DribbleFSM
     // the dribble tactic config
     TbotsProto::DribbleTacticConfig dribble_tactic_config;
     Point continuous_dribbling_start_point;
+    static constexpr double INTERCEPTION_POINT_ADJUSTMENT = 0.05015;
```

Comment:
```suggestion
    static constexpr double INTERCEPTION_POINT_ADJUSTMENT_M = 0.05015;
```

As a convention, we encode the units into the name of constants themselves